catroidTest/src/org/catrobat/catroid/uitest/formulaeditor/FormulaEditorListFragmentTest.java

```diff
@@ -0,0 +1,123 @@
+			R.string.formula_editor_function_arcsin, R.string.formula_editor_function_arccos,
+			R.string.formula_editor_function_arctan, R.string.formula_editor_function_exp,
+			R.string.formula_editor_function_floor, R.string.formula_editor_function_ceil,
+			R.string.formula_editor_function_max, R.string.formula_editor_function_min,
+			R.string.formula_editor_function_length, R.string.formula_editor_function_letter, R.string.formula_editor_function_join,
+			R.string.formula_editor_function_number_of_items, R.string.formula_editor_function_list_item, R.string.formula_editor_function_contains };
+
+	private static final int[] FUNCTIONS_PARAMETERS = { R.string.formula_editor_function_sin_parameter,
+			R.string.formula_editor_function_cos_parameter, R.string.formula_editor_function_tan_parameter,
+			R.string.formula_editor_function_ln_parameter, R.string.formula_editor_function_log_parameter,
+			R.string.formula_editor_function_pi_parameter, R.string.formula_editor_function_sqrt_parameter,
+			R.string.formula_editor_function_rand_parameter, R.string.formula_editor_function_abs_parameter,
+			R.string.formula_editor_function_round_parameter, R.string.formula_editor_function_mod_parameter,
+			R.string.formula_editor_function_arcsin_parameter, R.string.formula_editor_function_arccos_parameter,
+			R.string.formula_editor_function_arctan_parameter, R.string.formula_editor_function_exp_parameter,
+			R.string.formula_editor_function_floor_parameter, R.string.formula_editor_function_ceil_parameter,
+			R.string.formula_editor_function_max_parameter, R.string.formula_editor_function_min_parameter,
+			R.string.formula_editor_function_length_parameter, R.string.formula_editor_function_letter_parameter,
+			R.string.formula_editor_function_join_parameter, R.string.formula_editor_function_number_of_items_parameter,
+			R.string.formula_editor_function_list_item_parameter, R.string.formula_editor_function_contains_parameter };
+
+	public FormulaEditorListFragmentTest() {
+		super(MainMenuActivity.class);
+	}
+
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+		createProject(UiTestUtils.DEFAULT_TEST_PROJECT_NAME);
+		UiTestUtils.getIntoScriptActivityFromMainMenu(solo);
+	}
+
+	private void createProject(String projectName) throws InterruptedException {
+		project = new Project(null, projectName);
+		firstSprite = new Sprite("firstSprite");
+		Sprite secondSprite = new Sprite("secondSprite");
+
+		project.addSprite(firstSprite);
+		project.addSprite(secondSprite);
+
+		ProjectManager.getInstance().setProject(project);
+		ProjectManager.getInstance().setCurrentSprite(firstSprite);
+
+		Script startScript1 = new StartScript();
+		Script startScript2 = new StartScript();
+		Brick changeBrick = new ChangeSizeByNBrick(0);
+		Brick glideToBrick = new GlideToBrick(0, 0, 0);
+
+		firstSprite.addScript(startScript1);
+		secondSprite.addScript(startScript2);
+		startScript1.addBrick(changeBrick);
+		startScript2.addBrick(glideToBrick);
```

Comment:
Keep the code short, remove every code line which isn't necessary. Like bricks or the seconds sprite.